src/modules/constants/entities/constants-spi.entity.ts

```diff
@@ -16,7 +16,7 @@ export class ConstantSpiEntity {
       from: (value: string) => value.toString(),
     },
   })
-  @ApiProperty({ examples: ['A', 'B', 'C', 'Quality of Product', 'Percentage of Cover'] })
+  @ApiProperty({ example: 'A, B, C, Quality of Product, Percentage of Cover' })
```

Comment:
We'll have to pick only one of these to be our example if we can't use an array. If we think it's important to list all of these then we could pick one as an `example` and list the others in the `description` instead

Reply:
Happy with that approach, @avaitonis what are your thoughts on above pleasE?

Reply:
Have agreed with @avaitonis on the pushed change.